extras/lbryinc/client.go

```diff
@@ -13,11 +13,18 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
-// Client stores data about internal-apis call it is about to make.
+// Caller interface defines currently supported internal-apis methods.
+type Caller interface {
+	SetServerAddress(string)
+	GetServerAddress() string
+	UserMe() (ResponseData, error)
```

Comment:
`ResponseData` is a bad name. It should return `UserMeResponse` to conform to how we handle other clients with go. So that no fields of the response need to be parsed after the client has returned the response.